.gitignore

```diff
@@ -1,4 +1,4 @@
-/vendor/
-/build/
 *.swp
-
+/build/
+/composer.lock
```

Comment:
I was surprised that the composer.lock file got .gitignored at first because I remembered reading that it should be versioned in the composer documentation. Then I stumbled on this SO post and started appreciating some of the alternate perspectives.

https://stackoverflow.com/questions/12896780/should-composer-lock-be-committed-to-version-control

So is it better to ignore the .lock file in our case because we are building a library, whereas the .lock file should be committed in the pivot-libre repo because it is an app?